[Enhancement] Add HTTP RestFul interface to get QueryPlan by QueryID

chaoyli opened the pull request:
1. Doris is lack of get QueryPlan by QueryID, so I add it.
2. Insert is not be taken into consideration when been profiled.
   Now I add it into QueryDetailQueue to be collected.

morningman commented:
Hi @chaoyli,
I am sorting out all current http apis of FE, and I am going to standardize the incoming parameters, return values and other information of these apis.
Could you please wait a few days before I finish doing this?
The current API implementation is not standardized, we need a unified standard to design these APIs.

chaoyli committed:
[BUG] Using attachement strategy of brpc to send packet with big size.
BRPC send packet should serialize it first and then send it.
If we send one batch with big size, it will encounter a connection failed.
So we can use attachment strategy to bypass the problem and eliminate
the serialization cost.

chaoyli commented:
> Hi @chaoyli,
> I am sorting out all current http apis of FE, and I am going to standardize the incoming parameters, return values and other information of these apis.
> Could you please wait a few days before I finish doing this?
> The current API implementation is not standardized, we need a unified standard to design these APIs.

OK, I will close it temporary. After you finish the standardization, I will reopen it.
